src/apps/vpn/ui/screens/getHelp/giveFeedback/GiveFeedbackRadioDelegate.qml

```diff
@@ -44,8 +44,10 @@ RadioDelegate {
             anchors.fill: parent
             source: img
             color: VPNTheme.colors.blue
-            visible: radio.checked || radio.state === uiState.stateHovered
-                || radio.state === uiState.statePressed
+            //prevents iOS and Android from getting into a weird hover state
+            visible: radio.checked || radio.state === uiState.statePressed
+                     || ((Qt.platform.os !== "android" && Qt.platform.os !== "ios")
```

Comment:
Nit: This condition is pretty hard to ready now. Should we at least move `Qt.platform.os !== "android" && Qt.platform.os !== "ios"` to a variable like `isDesktop` or alike?